src/coreclr/inc/corinfo.h

```diff
@@ -1726,10 +1726,12 @@ struct CORINFO_FIELD_INFO
 
 struct CORINFO_THREAD_STATIC_BLOCKS_INFO
 {
-    CORINFO_CONST_LOOKUP tlsIndex;
-    uint32_t offsetOfThreadLocalStoragePointer;
-    uint32_t offsetOfMaxThreadStaticBlocks;
-    uint32_t offsetOfThreadStaticBlocks;
+    CORINFO_CONST_LOOKUP tlsIndex;              // windows specific
+    uint32_t offsetOfThreadLocalStoragePointer; // windows specific
+    size_t offsetOfMaxThreadStaticBlocks;
+    size_t offsetOfThreadStaticBlocks;
+    size_t tlsGetAddrFtnPtr;                    // linux/x64 specific
+    size_t threadStaticsBaseOffset;             // linux/x64 specific
```

Comment:
It does not look right to me that this is called `BaseOffset`. This value is not an offset. It is an address.

The Linux ABI documentation calls it as `tls_index object` so we can call it the same so that it is easy to match, i.e. `void* tlsIndexObject; // address of tls_index object` 

Also, I would move `tlsGetAddrFtnPtr` and this one before `offsetOfMaxThreadStaticBlocks` since it is more logical place for them.

Reply:
done